src/routers/swap_router.ts

```diff
@@ -8,10 +8,16 @@ import { SwapService } from '../services/swap_service';
 export function createSwapRouter(swapService: SwapService): express.Router {
     const router = express.Router();
     const handlers = new SwapHandlers(swapService);
-    router.get('/', asyncHandler(SwapHandlers.rootAsync.bind(SwapHandlers)));
-    router.get('/quote', asyncHandler(handlers.getSwapQuoteAsync.bind(handlers)));
-    router.get('/tokens', asyncHandler(handlers.getSwapTokensAsync.bind(handlers)));
-    router.get('/prices', asyncHandler(handlers.getTokenPricesAsync.bind(handlers)));
-    router.get('/price', asyncHandler(handlers.getSwapPriceAsync.bind(handlers)));
+    router.get('/v0', asyncHandler(SwapHandlers.rootAsync.bind(SwapHandlers)));
+    router.get('/v0/quote', asyncHandler(handlers.getSwapQuoteV0Async.bind(handlers)));
+    router.get('/v0/tokens', asyncHandler(handlers.getSwapTokensAsync.bind(handlers)));
+    router.get('/v0/prices', asyncHandler(handlers.getTokenPricesAsync.bind(handlers)));
+    router.get('/v0/price', asyncHandler(handlers.getSwapPriceV0Async.bind(handlers)));
```

Comment:
Why not have all the handlers take a `SwapVersion` as the first parameter and just do like `handlers.getSwapPriceAsync.bind(handlers, SwapVersion.V0)` so there isn't as much code dupe.